mlflow/models/python_api.py

```diff
@@ -0,0 +1,196 @@
+import json
+import os
+from io import StringIO
+from typing import Any, Dict, List, Optional, Union
+
+from mlflow.exceptions import MlflowException
+from mlflow.models.flavor_backend_registry import get_flavor_backend
+from mlflow.utils import env_manager as _EnvManager
+from mlflow.utils.file_utils import TempDir
+
+
+def build_docker(
```

Comment:
No change from the original definition in `mlflow/models/__init__.py`. Just moved to avoid complicating `__init__.py`.